app/src/docs/v1.api-spec.yaml

```diff
@@ -469,14 +470,14 @@ paths:
     put:
       summary: Replaces tags of an object
       description: >-
-        Replace the existing tags of an object with the given tagset. Multiple
-        comma separated Key/Value pairs can be provided in the query.
-      operationId: replaceTags
+        Replace the existing tags of an object with the given tagsets. Multiple
```

Comment:
It's a minor point but we probably want to stick with the S3 language which describes TagSet as a `tag-set` (singular) which is an array of tags. so a single k/v pair should be called a 'tag'. 
https://docs.amazonaws.cn/en_us/AmazonS3/latest/API/API_PutObjectTagging.html

here you could say: 'Replace the existing **tag-set** of an object with the given **tags**. Multiple Key/Value **tag** pairs can be provided in the query.'

Reply:
Fixed